packages/web3-state/src/Token.ts

```diff
@@ -175,4 +194,12 @@ export class TokenState<ChainId, SchemaType> implements Web3TokenState<ChainId,
         this.blockOrUnblockToken(address, token, 'block')
         this.addOrRemoveToken(address, token, 'add')
     }
+
+    async createFungibleToken(chainId: ChainId, address: string, token?: FungibleToken<ChainId, SchemaType>) {
+        return token
+    }
+
+    async createNonFungibleToken(chainId: ChainId, address: string, token?: NonFungibleToken<ChainId, SchemaType>) {
+        return token
```

Comment:
```suggestion
        throw new Error('Not implemented yet.')
```

Reply:
It's better to not implement `createNonFungibleToken` for `TokenState` in `packages/web3-state/src/Token.ts`